src/interface/src/app/treatments/treatments-tab/treatments-tab.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component } from '@angular/core';
+import { CommonModule, NgIf, NgFor } from '@angular/common';
+//TODO: fix path?
```

Comment:
:) yes!

Mind doing it on this pr? Should be an easy change.
Also the imports above should also be just `from '@styleguide'`